### PR TITLE
Add support for Netlify deployments

### PR DESCRIPTION
**Pull request by hectcastro**
This change set includes a variety of different changes to prepare the SPA for deployment via Netlify.

- [x] Connect Netlify URL to `population-map.districtbuilder.org` domain
- [x] Reset the main deployment branch in the Netlify configuration to `master`

Fixes https://github.com/azavea/operations/issues/466

---

**Testing**

- Ensure `npm run build` output contains no linter warnings
- Ensure the [application](https://deploy-preview-17--db-population-map.netlify.app/) functions as expected when served via Netlify

### CODE REVIEW

**Comment by jfrankl** [comment]
Sorry, I just merged a commit that caused these two conflicts. I can address these later tonight, or you're welcome to do it if you want. My version of `netlify.toml` can be totally replaced by yours, and I'm hoping the `MapboxGLMap.js` isn't significant.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "underover-districts",
+  "name": "undercover-districts",
```
Comment (hectcastro) [inline package.json]: Not sure, but I thought this was a typo.

File: src/components/MapboxGLMap.js [hunk 2]
```diff
@@ -435,7 +453,8 @@ const MapboxGLMap = () => {
             </p>
             <p>
               Use this map to see which districts are over-or-under populated.
-              For more information, read the corresponding <a href="#">blog</a>.
+              For more information, read the corresponding
+              <a href="https://www.azavea.com/blog/">blog</a>.
```
Comment (hectcastro) [inline src/components/MapboxGLMap.js]: I believe that this line needs to be updated once the blog post is published.

**Review by hectcastro — COMMENTED** [review]
Merge conflicts should be gone now—I rebased this branch with your changes on `master`.

File: package.json (hunk 1 above)
Comment (jfrankl) [reply package.json]: Ha, it wasn't a typo: it was `underover` as in **Under**population, **over**population. But I see why you thought it was.

I will rename this to match the repo name: `us-cd-population-map`. I think that makes more sense. `underover` was a placeholder from early in the project.

**Review by jfrankl — APPROVED** [review]
Works great. Thank you for making this happen on short notice.

**Comment by hectcastro** [comment]
Np. The site should be up on https://population-map.districtbuilder.org/ now.

**Pull request merged**